### PR TITLE
690

### PR DESCRIPTION
**Pull request by FrancescoVassalli**
address #690

### CODE REVIEW

**Comment by cjbbb** [comment]
I will test it and make a judgement as soon as possible. Some cases like "phone number" maybe can be used for all companies.

**Comment by FrancescoVassalli** [comment]
> I will test it and make a judgement as soon as possible. Some cases like "phone number" maybe can be used for all companies.

I don't understand your point? It is checking if the company name is in the answer you probably don't want to reuse those answers

File: src/ai_hawk/linkedIn_easy_applier.py [hunk 1]
```diff
@@ -844,10 +857,9 @@ def _save_questions_to_json(self, question_data: dict) -> None:
                     logger.error("JSON decoding failed")
                     data = []
 
-                # Use the new function to check for existing questions
-                question_exists = self._question_exists_in_data(question_data['question'], data)
+                should_be_saved: bool = not question_already_exists_in_data(question_data['question'], data) or self.answer_contians_company_name(question_data['answer'])
```
Comment (sloganking) [inline src/ai_hawk/linkedIn_easy_applier.py:860]: shouldn't this be AND instead of OR? And shouldn't it be NOT self.answer_contians_company_name?

Comment (FrancescoVassalli) [reply src/ai_hawk/linkedIn_easy_applier.py]: Yes fixed

**Comment by feder-cr** [comment]
@FrancescoVassalli join on telegram group

**Pull request merged**